includes/class-donations.php

```diff
@@ -103,6 +114,24 @@ public static function get_donation_settings() {
 
 		$settings['tiered'] = (bool) $product->get_meta( self::DONATION_TIERED_META, true );
 
+		// Add the product IDs for each frequency.
+		foreach ( $product->get_children() as $child_id ) {
+			$child_product = wc_get_product( $child_id );
+			if ( ! $child_product ) {
+				continue;
+			}
+
+			if ( 'subscription' === $child_product->get_type() ) {
+				if ( 'year' === $child_product->get_meta( '_subscription_period', true ) ) {
+					$settings['products']['year'] = $child_id;
+				} else {
+					$settings['products']['month'] = $child_id;
+				}
+			} else {
+				$settings['products']['once'] = $child_id;
```

Comment:
Is there a scenario where the grouped product is edited directly in WooCommerce (i.e. not by using Newspack's Wizard) and we end up with unexpected child products? Based on this risk, wondering if we should avoid catch-all `else` statements, both for determining the `once` product and the `month`?

Reply:
Good call. I've added better checks in 1cecd64 to more accurately determine whether a product should be included.